Adding initial CentOS doc for installing dependencies and WHLs on ROCm

parallelo opened the pull request:
Placeholder for some ROCm-specific docs on how to install TF via WHL file on CentOS.  Will need a few rounds of updates depending on what the DevOps team provides us.  

PS - Please don't merge yet.

poznano-amd committed:
Adding initial CentOS doc for installing dependencies and WHLs on ROCm

ROCm-Apps-Test commented:
Can one of the admins verify this patch?

mvermeulen commented:
A question on python versions combined with RHEL 8.2:
- The default python version in RHEL 8 is python 3.8 (http://distrowatch.org/table.php?distribution=redhat)
- The first manylinux2010 package we provide from repo.radeon.com is likely to be python 3.6 (which is also has python 3.6)
The first item above might be reason to list it as python 3.8 and the later item is a reason to list it as python 3.6.

So I am wondering if we might either make it explict to may sure python 3.6 is installed on the system first - and make these instructions one for "running 3.6 python whl package on RHEL 8.2" or instructions whose example is RHEL 7.8 which is also target audience for manylinux2010 whl package?  If the latter, then we have separate section on installing from PyPI?

stevenireeves commented:
We will just advertise this as the directions for RHEL as well?

parallelo commented:
Yep, very good points.  Seems like RHEL/CentOS 7 in the docs would make more sense in this case.

parallelo commented:
Pros/cons for each option.  Let's chat in the meeting this morning.

poznano-amd committed:
Misc updates to the CentOS docs

- Switched to CentOS 7.8
- Updated ROCm package list
- Fixed pip command
- Added a quick sanity test
- Tested using a docker image based on `rocm/dev-centos-7:3.7`

A final update is still needed for the public path to the manylinux WHLs.

parallelo commented:
FYI - Just updated the documentation with this [commit](https://github.com/ROCmSoftwarePlatform/tensorflow-upstream/pull/1079/commits/41de54632ca341947307000a50679cfbc89c5df1).  Initial sanity testing (using the listed steps) looks okay on CentOS 7.8.

parallelo commented:
Still need to add the final location of the public WHL.

poznano-amd committed:
Update tensorflow-install-basic-centos.md

stevenireeves committed:
RHEL Docs
